src/org/rascalmpl/library/lang/rascal/tests/library/lang/html/IOTests.rsc

```diff
@@ -12,3 +13,22 @@ test bool partialDocumentsAreCompletedOnRead()
 
 test bool partialDocumentsAreCompletedOnWrite() 
     = /html/ := writeHTMLString(p([text("paragraaf")]));
+
+test bool originTrackingElements() {
```

Comment:
good for adding tests!

could we maybe do a small xml file, and read back the source location, and check that it indeed points to the right place?